index.js

```diff
@@ -20,7 +19,6 @@ if (!$mapHas) {
 		// `Map` does not have a `has` method
 		return false;
 	};
-	return;
```

Comment:
Even if you were to get rid of these `return`s, this is not it; you need to convert the rest of the file into `else` bodies.